data/tekton-pipelines/okd/windows11-installer.yaml

```diff
@@ -0,0 +1,449 @@
+                      "requests": {
+                        "storage": "30Gi"
+                      }
+                    }
+                  },
+                  "source": {
+                    "blank": {}
+                  }
+                }
+              }
+        - name: deleteDisks
+          value: 'true'
+        - name: disks
+          value:
+            - '{"name": "installcdrom", "cdrom": {"bus": "sata"}, "bootOrder": 2}'
+            - '{"name": "rootdisk", "disk": {"bus": "virtio"}, "bootOrder": 1}'
+            - '{"name": "virtiocontainerdisk", "cdrom": {"bus": "sata"}}'
+            - '{"name": "sysprep", "cdrom": {"bus": "sata"}}'
+        - name: deleteVolumes
+          value: 'true'
+        - name: volumes
+          value:
+            - >-
+              {"name": "installcdrom", "dataVolume": {"name":
+              "${NAME}-installcdrom"}}
+            - '{"name": "rootdisk", "dataVolume": {"name": "${NAME}-rootdisk"}}'
+            - >-
+              {"name": "virtiocontainerdisk", "containerDisk": {"image":
+              "$(params.virtioContainerDiskName)"}}
+            - >-
+              {"name": "sysprep", "sysprep": {"configMap": {"name":
+              "$(params.autounattendConfigMapName)"}}}
+        - name: deleteTemplateParameters
+          value: 'true'
+        - name: templateParameters
+          value:
+            - >-
+              {"name": "NAME", "description": "VM name", "generate":
+              "expression", "from": "windows-[a-z0-9]{6}"}
+      runAfter:
+        - copy-template
+        - modify-windows-iso-file
+      taskRef:
+        kind: ClusterTask
+        name: modify-vm-template
+      timeout: 10m0s
+    - name: create-vm-from-template
+      params:
+        - name: templateName
+          value: $(tasks.modify-vm-template.results.name)
+        - name: runStrategy
+          value: RerunOnFailure
+        - name: startVM
+          value: 'true'
+      runAfter:
+        - modify-vm-template
+      taskRef:
+        kind: ClusterTask
+        name: create-vm-from-template
+    - name: wait-for-vmi-status
+      params:
+        - name: vmiName
+          value: $(tasks.create-vm-from-template.results.name)
+        - name: successCondition
+          value: status.phase == Succeeded
+        - name: failureCondition
+          value: 'status.phase in (Failed, Unknown)'
+      runAfter:
+        - create-vm-from-template
+      taskRef:
+        kind: ClusterTask
+        name: wait-for-vmi-status
+      timeout: 2h0m0s
+    - name: create-base-dv
+      params:
+        - name: manifest
+          value: |
+            apiVersion: cdi.kubevirt.io/v1beta1
+            kind: DataVolume
+            metadata:
+              name: $(params.baseDvName)
+              namespace: $(params.baseDvNamespace)
+              annotations:
+                cdi.kubevirt.io/storage.bind.immediate.requested: "true"
+                cdi.kubevirt.io/storage.deleteAfterCompletion: "false"
```

Comment:
Do we still need to disable GC?

Reply:
the https://github.com/kubevirt/kubevirt-tekton-tasks/pull/204 is not yet merged and released. After it will be merged and released, I will update this PR.